packages/sync-actions/src/products.js

```diff
@@ -19,7 +19,8 @@ const actionGroups = [
   'references',
   'prices',
   'pricesCustom',
-  'attributes',
```

Comment:
while it makes alot of sense to rename it, i would avoid renaming `attributes` to `productAttributes`, cause that could be breaking change which require major upgrade.

there is multi ways to introduce it without breaking, the easiest would be keep it and only introduce `productAttributes`